Makefile

```diff
@@ -1,4 +1,6 @@
-URL := http://localhost:1313
+URL ?= http://localhost:1313
+PORT ?= 1313
+BIND ?= 0.0.0.0
```

Comment:
Default was "localhost", I would keep it at that and only change if the URL has been explicitly set.

Not that it's a huge security risk, but just a though..

From the hugo docs:
> --bind string            interface to which the server will bind (default "127.0.0.1")